docs/reference/migration/migrate_7_13.asciidoc

```diff
@@ -288,8 +288,8 @@ errors in 8.0.0.
 The `index.indexing.slowlog.level` and `index.search.slowlog.level` index
 settings are now deprecated. You use these setting to set the logging level for
 the search and indexing slow logs. To reproduce similar results, use the
-respective `index.*.slowlog.threshold.index.debug` and
-`index.*.slowlog.threshold.index.trace` index settings instead.
+respective `index.*.slowlog.threshold.*.debug` and
+`index.*.slowlog.threshold.*.trace` index settings instead.
```

Comment:
I think you are right that this is not exactly clear.
But at the same time, when reading this I don't think `*` will make it better.
perhaps we could reword this without *:
```
respective `index.index.slowlog.threshold.index.debug` and
`index.indexing.slowlog.threshold.index.trace` index settings instead.
The same applies to `index.search.slowlog.level` which should be replaced with 
`index.search.slowlog.threshold.query.{level}`

Reply:
Sounds good, updated as suggested.